src/components/character/character.jsx

```diff
@@ -53,19 +53,18 @@ const Character = React.createClass(({
         }
 
         newCharacter.guid = generateGuid();
-
+        //fieldsToSave not technically an array so we call the array method and apply the fields as though they were elements in an array
```

Comment:
I'm using the forEach method from an empty array because fieldsToSave are an HTMLCollection and, don't have the forEach function. I pass in fieldsToSave as the object to be iterated over in the forEach method instead.

I could either of these:
Array.prototype.forEach.call(els, function(el) {
});

 Or
[].forEach.call(els, function (el) {...});